tests/zenko_tests/node_tests/package.json

```diff
@@ -12,7 +12,7 @@
   ],
```

Comment:
Any reason to change the file permissions from 644 to 755?

Reply:
good find, i didn't notice that